3drepo/3drepo.io
# Issue 5149 User Permission - Warning modal when changing permissions

This fixes #5149

#### Description
Changing a user permissions prompts a modal if the flag is set to false. 
It is possible to toggle the flag at any point from the pages where you can change permissions

#### Test cases
- Try to change the permissions from:
   - Users tab in teamspace page;
   - Project permission tabs in project page (both 1 or a batch);
   - Users permissions tabs in project page; (both 1 or a batch);

- Make sure setting the modal flag to false will not trigger the modal
- Make sure, after logging out and logging back in, the flag is set to false
